runtime/src/test/java/org/eclipse/xpanse/runtime/testContainers/ZitadelTestContainer.java

```diff
@@ -0,0 +1,42 @@
+package org.eclipse.xpanse.runtime.testContainers;
+
+import java.io.File;
+import org.junit.jupiter.api.BeforeAll;
+import org.testcontainers.containers.DockerComposeContainer;
+
+public class ZitadelTestContainer {
+
+    private static final String DOCKER_COMPOSE_PATH =
+            new File("src/test/resources/docker-compose.yml").getAbsolutePath();
+    private static DockerComposeContainer<?> zitadelComposeContainer;
+
+    static {
+        if (zitadelComposeContainer == null) {
+            File dockerComposeFile = new File(DOCKER_COMPOSE_PATH);
+            System.out.println(
```

Comment:
replace all log lines to use a proper logger. 
You can annotate the class with '@Sl4j' and then use the logger.